README.md

```diff
@@ -80,12 +80,15 @@ zone.  This role has a trust relationship with the users account.
 | [aws_route53_record.crossfeed_staging_A](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
 | [aws_route53_record.crossfeed_staging_AAAA](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
 | [aws_route53_record.crossfeed_staging_CNAME](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
```

Comment:
I will globally apply `terraform-docs` when the new PR is created.

Reply:
I'm not following what the conflict is. What needs to be changed in the new PR?

Reply:
Yes, I believe that is the only change needed here.

@jsf9k For the terraform apply to succeed, you might have to first manually destroy `aws_route53_record.crossfeed_staging_CNAME` before it will let you apply `aws_route53_record.crossfeed_staging_CNAME2` since they are for the same FQDN.  Or you could `terraform import` it into `aws_route53_record.crossfeed_staging_CNAME2` too.

Reply:
I have removed the line, but I need access to push my branch up, could you please do me the honor :-) ?

Reply:
@dav3r I think the reason the Terraform doesn't apply is because `aws_route53_record.crossfeed_staging_CNAME2` conflicts with `aws_route53_record.crossfeed_staging_A`.  Those two records can't coexist.

Reply:
Hi @jsf9k, that sounds reasonable, how do you suggest I proceed?

Reply:
@actualeyes, one of the records has to go.  I'm not sure which one.  Maybe @epicfaace knows?

Reply:
> @dav3r I think the reason the Terraform doesn't apply is because `aws_route53_record.crossfeed_staging_CNAME2` conflicts with `aws_route53_record.crossfeed_staging_A`. Those two records can't coexist.

Ah, I totally missed that.  Good call.

Reply:
Just for my context, why do those records conflict ?

Reply:
[This link](https://serverfault.com/questions/1049792/what-happens-if-i-have-both-a-record-and-cname-record-on-my-domain) explains.

Reply:
Because they both have the same value for their name:
```
name    = "staging.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
```

Reply:
```
27cc0191 (Hillary          2020-09-01 11:16:48 -0400 200) resource "aws_route53_record" "crossfeed_staging_A" {
27cc0191 (Hillary          2020-09-01 11:16:48 -0400 201)   provider = aws.route53resourcechange
27cc0191 (Hillary          2020-09-01 11:16:48 -0400 202) 
27cc0191 (Hillary          2020-09-01 11:16:48 -0400 203)   alias {
27cc0191 (Hillary          2020-09-01 11:16:48 -0400 204)     name                   = "do8d5i2re25cf.cloudfront.net."
422e2da2 (Hillary          2020-09-02 12:05:05 -0400 205)     zone_id                = var.cloudfront_zone_id
27cc0191 (Hillary          2020-09-01 11:16:48 -0400 206)     evaluate_target_health = false
27cc0191 (Hillary          2020-09-01 11:16:48 -0400 207)   }
27cc0191 (Hillary          2020-09-01 11:16:48 -0400 208)   name    = "staging.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
27cc0191 (Hillary          2020-09-01 11:16:48 -0400 209)   type    = "A"
27cc0191 (Hillary          2020-09-01 11:16:48 -0400 210)   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
27cc0191 (Hillary          2020-09-01 11:16:48 -0400 211) }
27cc0191 (Hillary          2020-09-01 11:16:48 -0400 212) 
```

```
commit 27cc01916d5781a1e84a319bf172aec680469f78
Author: Hillary <hillaryaj@gmail.com>
Date:   Tue Sep 1 11:16:48 2020 -0400

    Add Crossfeed entries

diff --git a/route53_crossfeed_app.tf b/route53_crossfeed_app.tf
new file mode 100644
index 0000000..d305f1f
```

Looks like these were all assigned together when crossfeed was added.

Reply:
@actualeyes `aws_route53_record.crossfeed_staging_CNAME2` was added in this PR and it has the same value as the already existing `aws_route53_record.crossfeed_staging_A`. You cannot add a CNAME record for an already existing A record.

Reply:
That makes sense given the comments and references above. 
You'll have to forgive my ignorance as my role wasn't to make this change but ensure that it makes it through. 

So, it's likely best to focus on the objective:
We need to add the below 2 CNAMES for a project. 
- CNAME staging.api.crossfeed.cyber.dhs.gov crossfeed-stage-1792947306.us-gov-west-1.elb.amazonaws.com
- CNAME staging.crossfeed.cyber.dhs.gov crossfeed-stage-1792947306.us-gov-west-1.elb.amazonaws.com
Since CNAME2 is already an A record is it possible to just delete CNAME2 since it's already there assuming that the other two records are for the above links? 

Please advise.

Reply:
@epicfaace and @actualeyes, were you able to determine the best way to remedy the DNS record collision?  Is there a new PR forthcoming?

Reply:
Yeah, we need 1) to remove the existing A / ALIAS records and also 2) provision new ACM certs for staging-lz. domains and add the CNAMEs for verification for those certs. New PR forthcoming

Reply:
The new PR is #78.